local-dev/templates/rawls.conf.ctmpl

```diff
@@ -401,6 +401,10 @@ multiCloudWorkspaces {
       "AKS_AUTOSCALING_MAX": "100"
       "AKS_MACHINE_TYPE": "Standard_D4as_v5"
     }
+    costSavingLandingZoneParameters = {
+      "AKS_COST_SAVING_SPOT_NODES_ENABLED": "true"
+      "AKS_COST_SAVING_VPA_ENABLED": "true"
+    }
```

Comment:
If you store `costSavingLandingZoneParameters` in rawls.conf, you'll need to make a corresponding change to the config in terra-helmfile:
https://github.com/broadinstitute/terra-helmfile/blob/a024f3908c519139ee55dbc6695f62522f53af3c/charts/rawls/templates/config/_conf.tpl#L430. The `local-dev/templates/rawls.conf.ctmpl` file is only used when running Rawls locally.

Alternately, you could put this stanza in [core/src/main/resources/reference.conf](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/resources/reference.conf) and leave it out of rawls.conf altogether.

**reference.conf**: the default values for any setting. Requires a full rebuild/redeploy of Rawls to change, since it's right in the codebase. But, these values are designed to be overridden by values in rawls.conf, so you can effectively change them by overriding. 

**rawls.conf**: the environment-specific values for any setting. Can be changed in k8s "live", with a simple rolling restart, no rebuild or redeploy needed. 

If the `costSavingLandingZoneParameters` config stanza doesn't change between environments (dev vs. prod), and doesn't change frequently, it's a pretty good candidate to store in reference.conf. 

See https://github.com/lightbend/config?tab=readme-ov-file#standard-behavior for understanding how the Typesafe Config library layers these config files. When reading that page, substitute "rawls.conf" anywhere you see "application.conf".